### PR TITLE
feat: Introduce 'mc batch' command

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cmd/batch-trace.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	fatalIf(err, "Unable to initialize admin connection.")
+
+	ctxt, cancel := context.WithCancel(globalContext)
+	defer cancel()
+
+	opts, e := tracingOpts(ctx, []string{"batch-replication"})
+	fatalIf(probe.NewError(e), "Unable to start tracing")
+
+	mopts := matchOpts{
+		funcNames: ctx.StringSlice("funcname"),
```
Comment (klauspost) [inline cmd/batch-trace.go:95]: Isn't `response-threshold` missing?

File: cmd/batch-create.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	Usage:        "display current job creatermation",
+	Action:       mainBatchCreate,
+	OnUsageError: onUsageError,
+	Before:       setGlobalsFromContext,
+	Flags:        globalFlags,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET
```
Comment (klauspost) [inline cmd/batch-create.go:39]: ... and file?

File: cmd/batch-trace.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/batch-trace.go:95]: That's part of traceOpts

**Comment by harshavardhana** [comment]
PTAL this CLI is finalized for now, the corresponding server PR is cherry-picked from the branch.

**Pull request merged**